cmd/internal/cli/search.go

```diff
@@ -12,28 +12,27 @@ package cli
 import (
 	"runtime"
 
-	"github.com/apptainer/apptainer/internal/pkg/client/oci"
-	"github.com/apptainer/apptainer/internal/pkg/util/uri"
-
 	"github.com/apptainer/apptainer/docs"
+	"github.com/apptainer/apptainer/internal/pkg/client/library"
 	"github.com/apptainer/apptainer/pkg/cmdline"
 	"github.com/apptainer/apptainer/pkg/sylog"
 	"github.com/spf13/cobra"
+	"github.com/sylabs/scs-library-client/client"
 )
 
 var (
-	// SearchRegistryURI holds the base URI to a Registry API instance
-	SearchRegistryURI string
+	// SearchLibraryURI holds the base URI to a Sylabs library API instance
```

Comment:
@DrDaveD library/registry API ?

Reply:
I removed 'Sylabs'.  The function does not support general registry searches, only library searches.